l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1274,6 +1299,15 @@ def importFatturaPA(self, cr, uid, ids, context=None):
                 xml_string = fatturapa_attachment.datas.decode('base64')
             xml_string = self.remove_xades_sign(xml_string)
             xml_string = self.strip_xml_content(xml_string)
+            # Fix Date format
+            # Date format must be YYYY-MM-DD
+            # DateType in XSD implements
+            # restrict xs:date minInclusive YYYY-MM-DD  and date should be
+            # in others not parsable ISO 8601:2004 formats like:
+            # YYYY-MM-DDT00:00:00+00:00
+            # YYYY-MM-DD+00:00
+            # YYYY-MM-DDT00:00:00Z
```

Comment:
grazie verifico..




2015-06-05 9:41 GMT+02:00 Lorenzo Battistini <notifications@github.com>:

> In l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py
> <https://github.com/eLBati/l10n-italy/pull/61#discussion_r31793002>:
>
> > @@ -1274,6 +1299,15 @@ def importFatturaPA(self, cr, uid, ids, context=None):
> >                  xml_string = fatturapa_attachment.datas.decode('base64')
> >              xml_string = self.remove_xades_sign(xml_string)
> >              xml_string = self.strip_xml_content(xml_string)
> > +            # Fix Date format
> > +            # Date format must be YYYY-MM-DD
> > +            # DateType in XSD implements
> > +            # restrict xs:date minInclusive YYYY-MM-DD  and date should be
> > +            # in others not parsable ISO 8601:2004 formats like:
> > +            # YYYY-MM-DDT00:00:00+00:00
> > +            # YYYY-MM-DD+00:00
> > +            # YYYY-MM-DDT00:00:00Z
>
> Caricando il file IT05979361218_007 su
> http://sdi.fatturapa.gov.it/SdI2FatturaPAWeb/AccediAlServizioAction.do?pagina=controlla_fattura
> ottengo
>
> File not compliant with the format
> Check riga: 85 - colonna: 89  '2015-06-03T10:21:58+00:00' is not a valid value for 'date'.  Check that the file transmitted complies with the format expected   00200
> File not compliant with the format
> Check riga: 85 - colonna: 89  The value '2015-06-03T10:21:58+00:00' of element 'DataScadenzaPagamento' is not valid.    Check that the file transmitted complies with the format expected   00200
> File not compliant with the format
> Check riga: 91 - colonna: 77  '2015-06-04+1:00' is not a valid value for 'date'.    Check that the file transmitted complies with the format expected   00200
> File not compliant with the format
> Check riga: 91 - colonna: 77  The value '2015-06-04+1:00' of element 'DataDecorrenzaPenale' is not valid.   Check that the file transmitted complies with the format expected   00200
>
> Quindi direi che il file non è da accettare
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eLBati/l10n-italy/pull/61/files#r31793002>.
>



-- 
Best Regards
Alessio Gerace


*   Think Studio*
     +39 3396911693
     www.thinkstudio.it